packages/@sanity/default-layout/src/navbar/Navbar.tsx

```diff
@@ -236,22 +257,7 @@ export const Navbar = memo(function Navbar(props: NavbarProps) {
           )}
 
           <LegacyLayerProvider zOffset="navbarPopover">
-            <Tooltip
-              portal
-              scheme="light"
-              content={
-                <Box padding={2}>
-                  {canCreateSome ? (
-                    <Text size={1}>Create new document</Text>
-                  ) : (
-                    <InsufficientPermissionsMessage
-                      currentUser={currentUser}
-                      operationLabel="create any document"
-                    />
-                  )}
-                </Box>
-              }
-            >
+            <Tooltip portal scheme="light" content={<Box padding={2}>{createTooltipContent}</Box>}>
               <SpacingBox marginRight={shouldRender.brandingCenter ? undefined : 2}>
                 <Button
                   aria-label="Create new document"
```

Comment:
Should it be added here that it is disabled if the schema is empty? `disabled={!canCreateSome || noDocumentOptions}`

Reply:
Good idea, I thought about that too! But it seems like the `canCreateSome` condition is taking care of disabling the button for both permission and empty schema situations. So I don't think it is necessary to add that! 🙂